osc-server/src/main/java/org/osc/core/broker/service/tasks/conformance/openstack/deploymentspec/DSUpdateOrDeleteMetaTask.java

```diff
@@ -286,6 +283,13 @@ private void conformToHostsSelection(Collection<String> selectedHosts,
                 }
             }
         }
+
+        for (DistributedApplianceInstance dai : daisToDelete) {
+            // Remove any extra sva/DAI
+            log.info("Removing DAI/SVA: " + dai.getName() + " for host: " + dai.getHostName());
```

Comment:
nitpick but use slf4j parameterized logging. Going forward we want to use this patttern for logging

logger.info("Removing DAI/SVA: {} for host: {}", dai.getName(), dai.getHostName());